media-libs/libplacebo/libplacebo-2.72.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_REQ_USE="xml"
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+
+if [[ "${PV}" == "9999" ]]; then
+	EGIT_REPO_URI="https://code.videolan.org/videolan/libplacebo.git"
+	inherit git-r3
+else
+	KEYWORDS="~amd64 ~ppc64 ~x86"
+	SRC_URI="https://code.videolan.org/videolan/libplacebo/-/archive/v${PV}/libplacebo-v${PV}.tar.gz"
+	S="${WORKDIR}/${PN}-v${PV}"
+fi
+
+inherit meson multilib-minimal python-any-r1
+
+DESCRIPTION="Reusable library for GPU-accelerated image processing primitives"
+HOMEPAGE="https://code.videolan.org/videolan/libplacebo"
+
+LICENSE="LGPL-2.1+"
+SLOT="0/$(ver_cut 2)" # libplacebo.so version
+IUSE="glslang lcms +opengl +shaderc test +vulkan"
+REQUIRED_USE="vulkan? ( || ( glslang shaderc ) )"
+RESTRICT="!test? ( test )"
+
+RDEPEND="glslang? ( dev-util/glslang[${MULTILIB_USEDEP}] )
+	lcms? ( media-libs/lcms:2[${MULTILIB_USEDEP}] )
+	opengl? ( media-libs/libepoxy[${MULTILIB_USEDEP}] )
+	shaderc? ( >=media-libs/shaderc-2017.2[${MULTILIB_USEDEP}] )
+	vulkan? ( media-libs/vulkan-loader[${MULTILIB_USEDEP}] )"
+DEPEND="${RDEPEND}"
+
+BDEPEND="virtual/pkgconfig
+	vulkan? (
+		${PYTHON_DEPS}
+		dev-util/vulkan-headers
+		$(python_gen_any_dep 'dev-python/mako[${PYTHON_USEDEP}]')
+	)"
+
+python_check_deps() {
+	has_version "dev-python/mako[${PYTHON_USEDEP}]"
```

Comment:
The dependency is behind `IUSE=vulkan`, so this check must be conditional as well. Due to the way this function works, I would recommend, *before* the `has_version` check:
`use vulkan || return 0`

Reply:
I took the code from https://dev.gentoo.org/~mgorny/python-guide/any.html#conditional-python-use which has the `use` check on `pkg_setup()` itself.

That being said I just realized it should be `has_version -b`

Reply:
Indeed, I didn't see that (also the *_pkg_setup condition).